ClickableTransparentOverlay/Overlay.cs

```diff
@@ -32,10 +33,22 @@ public abstract class Overlay : IDisposable
         private volatile CancellationTokenSource cancellationTokenSource;
         private volatile bool overlayIsReady;
 
+        #region Constructors
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Overlay"/> class.
+        /// </summary>
+        public Overlay() : this("")
+        {
+        }
+
         /// <summary>
         /// Initializes a new instance of the <see cref="Overlay"/> class.
         /// </summary>
-        public Overlay()
+        /// <param name="windowTitle">
+        /// Title of the window created by the overlay
+        /// </param>
+        public Overlay(string windowTitle) : this(windowTitle, false)
```

Comment:
it would be sad to not be able to make an application DPIAware and also set the window tile. maybe change this to
`public Overlay(bool DPIAware, string windowTitle)`

Reply:
actually nevermind. didn't read the full code-change.